packages/client/src/apis/rewards/RewardsExecuteAPI.ts

```diff
@@ -73,7 +73,7 @@ export class RewardsExecuteAPI extends BaseVertexAPI {
    *
    * @param params
    */
-  async claimAndStakeLiquidTokens(params: ClaimLiquidTokensParams) {
+  async claimAndStakeLiquidTokens(params: ExecuteClaimLiquidTokensParams) {
```

Comment:
should this be `ClaimAndStackLiquidTokensParams`? why do we need `Execute` prefix here?